static/css/components/work.less

```diff
@@ -130,10 +130,6 @@ textarea[name="work--description"] + h3 + .wmd-preview,
   content: attr(data-before);
 }
 
```

Comment:
I think you might have removed this h6 on accident? if so, can you restore it? @0simoo